pkg/resources/sku/sku.go

```diff
@@ -225,27 +241,31 @@ func (p *ProductConfig) Configure(obj metav1.Object) error {
 	}
 
 	configReplicas := p.resourceConfigs[obj.GetName()].Replicas
-	if p.sku.isUpdated && *replicas < configReplicas {
+	if *replicas < configReplicas {
```

Comment:
Hi @philbrookes wdyt about removing this check. As we're only updating on a sku.isUpdated true event this would never get updated if we wanted to move to a sku with a lower replica count.